FitVid/src/Pages/LikedVideos/LikedVideos.js

```diff
@@ -1,11 +1,29 @@
-import React from "react";
-import { Navbar, Footer, SideBar, Video, useLiked } from "./index";
-
+import React, { useEffect } from "react";
+import {
+  Navbar,
+  Footer,
+  SideBar,
```

Comment:
Suggestion : If you are facing trouble hiding the sidebar on Login Page , useLocation can help you find the pathname and using that you can hide the sidebar